Elevel/Persistence/Configurations/TestConfiguration.cs

```diff
@@ -17,6 +17,26 @@ public void Configure(EntityTypeBuilder<Test> builder)
             builder.HasOne(x => x.Speaking)
                 .WithMany(x => x.SpeakingTests)
                 .HasForeignKey(x => x.SpeakingId);
+
+            builder.HasOne(x => x.User)
+                .WithMany(x => x.Tests)
+                .HasForeignKey(x => x.UserId);
+
+            builder.HasOne(x => x.Hr)
+                .WithMany(x => x.Tests)
+                .HasForeignKey(x => x.HrId);
+
+            builder.HasOne(x => x.Coach)
+                .WithMany(x => x.Tests)
+                .HasForeignKey(x => x.CoachId);
+
+            builder.HasOne(x => x.Audition)
+                .WithMany(x => x.Tests)
+                .HasForeignKey(x => x.AuditionId);
+
+            builder.HasMany(x => x.TestQuestions)
```

Comment:
Does TestQuestion have one test?? if so it's better to support one 1 configuration like 33-35 rows in current class. HasOne - withMany - HasForeignKey

Reply:
Solved